spec/game_runner_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'spec_helper'
+require_relative '../game_runner.rb'
+
+RSpec.describe GameRunner do
+  def grid_template
+    <<~GRID
+   __ __ __
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+  |  |  |  |
+  |__|__|__|
+    GRID
+  end
+
+  describe '#run' do
+    it 'on start up, an empty grid is rendered on the screen' do
+      stdout = StringIO.new(' ')
+      @game_start = GameRunner.new(stdout)
```

Comment:
Why are these instance variables?

Reply:
I would like to be able to create a new instance of `GameRunner` only once and have it be accessible throughout the rest of my tests as `@game_start`.

Reply:
Tests should be independent from each other. We very often work with clients who want to share state between tests for some alleged performance gain, but can't say with certainty what they are testing in each test.

We want each test to be fully independent. That means in each test you will have to exercise the code until you get the desired output.